common/src/main/java/com/gregtechceu/gtceu/api/data/worldgen/generator/GeodeVeinGenerator.java

```diff
@@ -245,6 +245,11 @@ public VeinGenerator build() {
         return this;
     }
 
+    @Override
+    public VeinGenerator copy() {
+        return new GeodeVeinGenerator(this.geodeBlockSettings, this.geodeLayerSettings, this.geodeCrackSettings, this.usePotentialPlacementsChance, this.useAlternateLayer0Chance, this.placementsRequireLayer0Alternate, this.outerWallDistance, this.distributionPoints, this.pointOffset, this.minGenOffset, this.maxGenOffset, this.noiseMultiplier, this.invalidBlocksThreshold);
```

Comment:
I think splitting up the args into separate lines could help with readability if you want.